apps/tmbitwarden.sh

```diff
@@ -218,7 +268,12 @@ add_dockerprune
 
 msg_box "Bitwarden was sucessfully installed! Please visit $SUBDOMAIN to setup your account.
 
-After the account it setup, please disable user registration by running sudo bash $SCRIPTS/menu.sh and choose:
-Additional Apps --> Bitwarden Registration"
+After the account is registered, please disable user registration by running sudo bash $SCRIPTS/menu.sh and choose:
+Additional Apps --> Bitwarden Registration
+
+Some notes to the bitwarden service:
+to START Bitwarden, simply execute: 'systemctl start bitwarden'
+to STOP Bitwarden, simply execute: 'systemctl stop bitwarden'
+to RESTART Bitwarden, simply execute: 'systemctl restart bitwarden'"
```

Comment:
BTW: are you really sure, that restart works? since we haven't configured this in the service file...